.eslintrc

```diff
@@ -28,13 +28,15 @@
         "max-len": "off",
         "react/display-name" : "off",
         "sort-imports": "off",
+        // Most of the warnings arises from inline arrow functions in Formik.
+        // As long as we use Formik looks like we can't get rid of this warning.
+        "react/jsx-no-bind": "off",
```

Comment:
I would prefer not to turn this rule off.  It is a good rule, no?  For the formik warnings we can do a one-line disable with a eslint comment.

Reply:
Agree, this is a good rule. I was not sure if to disable it globally (since most of them related to Formik) or disable specifically. I didn't want to add many lines in the middle of the code. What do you say about disabling for specific files? It's a one line in the top of the file.

Reply:
I think it best to disable it line-by line, as we were doing.  Thus it is clear in the code and we don't risk disabling for cases that we should fix.